README.md

```diff
@@ -11,4 +11,41 @@ Supported formats:
    ```
     gradlew clean build
    ``` 
-3. Find the distribution in build/distributions.
\ No newline at end of file
+3. Find the distribution in build/distributions.
+
+## Transform XML report to SARIF report
+
+1. Navigate to the `parasoft-report-transformer-1.0.0\bin` directory.
+2. Run:
+   * For Windows:
+      ```cmd
+      XMLToSARIF.bat -i <path/to/input/XML/report> [-o <path/to/output/SARIF/report>] [-t <path/to/parasoft/tool/installation/or/Java/home/directory>]
+      ```
+   * For Linux:
+      ```shell
+      ./XMLToSARIF.sh -i <path/to/input/XML/report> [-o <path/to/output/SARIF/report>] [-t <path/to/parasoft/tool/installation/or/Java/home/directory>]
+      ```
+3. The SARIF report will be generated in the same path of XML report.
+
+Notes:
```

Comment:
I think we should make "Note" a single section with a proper name like **Features** because we will need to add an explanation for relative URIs soon, and it's more than just a note. Besides, we should add more information to the explanation of the checked list to avoid confusion for people who may not have knowledge about it. For example: What does "checked" mean? Where are the artifacts located in the SARIF report? How to apply this features to the trasformation?
What's your opinion? @yshi-parasoft @rwang-parasoft

Reply:
I agree.